data/pokemon.json

```diff
@@ -661,31 +715,37 @@
     "nickname": [
       "arty"
     ],
-    "tier": 5
+    "tier": 5,
+    "shiny": true
   },
   {
     "name": "zapdos",
     "nickname": [
       "zappy"
     ],
-    "tier": 5
+    "tier": 5,
+    "shiny": true
   },
   {
     "name": "moltres",
     "nickname": [
       "molty"
     ],
-    "tier": 5
+    "tier": 5,
+    "shiny": true
   },
   {
     "name": "dratini",
-    "tier": 1
+    "tier": 1,
+    "shiny": true
   },
   {
-    "name": "dragonair"
+    "name": "dragonair",
+    "shiny": true
```

Comment:
Can't be shiny as a raid boss (that we know, not that it's ever been a raid boss)